.travis.yml

```diff
@@ -43,8 +43,8 @@ install:
     - mkdir ibmtpm974 && pushd ibmtpm974 && tar axf ../ibmtpm974.tar.gz && pushd ./src && make
     - ./tpm_server &
     - popd && popd
-    - git clone https://github.com/01org/TPM2.0-TSS.git
-    - pushd TPM2.0-TSS
```

Comment:
I did the same change when fixing the Travis CI build on PR #701.

So probably you want to drop this commit and merge on top? Or I can re-spin my branch without that change, but it would create conflicts anyways.

Reply:
Merges make everything harder, ill fix up your branch after this.